tools.fsproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="FSharp.NET.Sdk;Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FSharp.Core" Version="4.1.*" />
+    <PackageReference Include="FSharp.NET.Sdk" Version="1.0.*" PrivateAssets="All" />
+  </ItemGroup>
```

Comment:
just call it `tools.proj` and leave only

```xml
<Project Sdk="Microsoft.NET.Sdk">
  <ItemGroup>
    <DotNetCliToolReference Include="dotnet-mergenupkg" Version="1.0.*" />
    <DotNetCliToolReference Include="dotnet-fssrgen" Version="3.3.*" />
  </ItemGroup>
</Project>
```

like that the sdk give the restore target, but no compile (is not needed)